Author: Sven Joachim <svenjoac@gmx.de>
Description: Omit -L part from ncurses5{w,}-config output
 "ncurses5-config --libs" includes a "-L" part which is not needed
 since the library is installed in a directory in the default linker
 search path.  The value is also wrong if ncurses-bin is of a
 different architecture than libncurses5-dev.
 .
 Upstream only omits the -L part if $libdir is /usr/lib.  Until there
 is a way to obtain the multiarch path without running
 dpkg-architecture our patch is not upstreamable.
Bug-Debian: http://bugs.debian.org/638281
Forwarded: not-needed
Last-Update: 2012-07-15

---
 misc/ncurses-config.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/misc/ncurses-config.in
+++ b/misc/ncurses-config.in
@@ -84,7 +84,7 @@
 ENDECHO
 		;;
 	--libs)
-		if test "$libdir" = /usr/lib
+		if true
 		then
 			LIBDIR=
 		else
